R/read_in_files.R

```diff
@@ -311,18 +334,25 @@ read_unit_document_map_data <- function(project_name,
     paths <- build_paths(project_name = project_name, data_path = data_path,
                                df_path = df_path, project_path = project_path,
                                data_frame_name = data_frame_name)
+    if (file.exists(paths[["data"]])){
 
-        if (file.exists(paths[["data"]])){
-
-             qcoder_unit_document_map <- readr::read_csv(file = paths[["data"]],
-                                      col_types = readr::cols(doc_path = "c",
+          qcoder_unit_document_map <- readr::read_csv(file = file_path,
```

Comment:
This looks like when we dealt with the merge conflicts we picked the wrong version ... look at the file=